python/lsst/resources/_resourceHandles/_baseResourceHandle.py

```diff
@@ -38,10 +38,10 @@ class CloseStatus(Enum):
 
 class ResourceHandleProtocol(Protocol, Generic[U]):
     """Defines the interface protocol that is compatible with children of
-    `BaseResourceHandle`.
+    `.BaseResourceHandle`.
```

Comment:
<rant>
Ugh, I see so much of this in our future.  I really wish Sphinx could just look at how we've imported symbols!
</rant>